src/screens/AccountUnlock.js

```diff
@@ -84,8 +84,26 @@ export class AccountUnlock extends React.PureComponent {
 								pin
 							);
 						}}
-						navigate={() => {
-							if (next === 'AccountDelete') {
+						navigate={async pin => {
+							if (next === 'AccountBiometric') {
```

Comment:
extract this block of code out of `render()` into a function, to make the JSX part code clean and slightly performance improvement.